core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -420,27 +424,28 @@ object ConsumerGroupCommand {
                                   .ofType(classOf[String])
     val options = parser.parse(args : _*)
 
+    val useOldConsumer = options.has(zkConnectOpt)
+
     val allConsumerGroupLevelOpts: Set[OptionSpec[_]] = Set(listOpt, describeOpt, deleteOpt)
 
     def checkArgs() {
       // check required args
-      if (options.has(newConsumerOpt)) {
+      if (useOldConsumer) {
+        CommandLineUtils.checkRequiredArgs(parser, options, zkConnectOpt)
+
+        if (options.has(bootstrapServerOpt))
+          CommandLineUtils.printUsageAndDie(parser, s"Option $bootstrapServerOpt is not valid with old consumer.")
+
+      } else {
         CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOpt)
 
         if (options.has(zkConnectOpt))
-          CommandLineUtils.printUsageAndDie(parser, s"Option $zkConnectOpt is not valid with $newConsumerOpt")
+          CommandLineUtils.printUsageAndDie(parser, s"Option $zkConnectOpt is only valid with old consumer.")
```

Comment:
This can't happen because the `else` only runs if `zkConnectOpt` is not present.